versioned/persist/adapter/src/main/java/org/projectnessie/versioned/persist/adapter/DatabaseAdapterConfig.java

```diff
@@ -111,6 +112,38 @@ default int getDefaultMaxKeyListSize() {
     return DEFAULT_MAX_KEY_LIST_SIZE;
```

Comment:
Do we need two database-level "hard" limit settings (wrt `getDefaultMaxKeyListEntitySize`)?

Reply:
But yea - having a configurable default for another configurable doesn't really make much sense. Removed the configurable defaults.

Reply:
Yes, that makes sense, but I guess that is controlled by the "non-default" parameters, right? My reading of the "default" parameters was that they define a hard limit for database-level entity sizes, which would normally be the same for any entity, I guess :thinking: